File: Python/Minor_Assignment_2/Q21.py
num=int (input("Enter a number: "))
i=1
flag=True
while num>1:
    if num % i !=0:
        flag=False
        break
    num=num//i
    i+=1
if flag:
    print("Factorial number")
else:
    print("Not a factorial number")

'''
Output---
Enter a number: 120
Factorial number
'''

File: PPWC/Minor_Assignment_5/Q21.c
#include<stdio.h>

int f(int n){
    while(--n>=0){
    printf("%d ",n-2);}
    return 1;
}
int main(){
    int (*p)(int)=f;
    (*p)(8);
    return 0;
}

/*
Output--
5 4 3 2 1 0 -1 -2
*/